src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.js

```diff
@@ -124,6 +125,7 @@ class BaseSidebarScreen extends Component {
                                 onPress={this.showCreateMenu}
                             />
                         </View>
+                        <KeyboardShortcutsModal />
```

Comment:
Question: Will this component be rendered each time BaseSidebarScreen render is triggered?

Reply:
This is the most expensive component of the app due to it is always mounted. We will have to think about putting the modal component here. 
we can also put this on higher level.

Reply:
Aha, good catch.